test/unit/itext.js

```diff
@@ -676,6 +676,46 @@
       assert.equal(style, '\n\t\t@font-face {\n\t\t\tfont-family: \'Plaster\';\n\t\t\tsrc: url(\'path-to-plaster-font-file\');\n\t\t}\n\t\t@font-face {\n\t\t\tfont-family: \'Engagement\';\n\t\t\tsrc: url(\'path-to-engagement-font-file\');\n\t\t}\n');
     });
 
+    QUnit.test('toSVGWithFontsInGroups', function(assert) {
+      var iText1 = new fabric.IText('test foo bar-baz\nqux', {
+        styles: {
+          0: {
+            0: { fill: '#112233' },
+            2: { stroke: '#223344', fontFamily: 'Engagement' },
+            3: { backgroundColor: '#00FF00' }
+          }
+        },
+        fontFamily: 'Plaster'
+      });
+      var iText2 = new fabric.IText('test foo bar-baz\nqux\n2', {
+        styles: {
+          0: {
+            0: { fill: '#112233', fontFamily: 'Plaster' },
+            2: { stroke: '#223344' },
+            3: { backgroundColor: '#00FF00' }
+          }
+        },
+        fontFamily: 'Engagement'
+      });
+      fabric.fontPaths = {
+        Engagement: 'path-to-engagement-font-file',
+        Plaster: 'path-to-plaster-font-file'
+      };
+      var subGroup = new fabric.Group([iText1]);
+      var group = new fabric.Group([subGroup, iText2]);
+      canvas.add(group);
+      assert.equal(typeof iText1.toSVG, 'function');
+      assert.equal(typeof iText2.toSVG, 'function');
+      var parser = new DOMParser();
+      var svgString = canvas.toSVG(),
+          doc = parser.parseFromString(svgString, 'image/svg+xml'),
+          style = doc.getElementsByTagName('style')[0].firstChild.data;
+      assert.equal(
+        style,
+        '\n\t\t@font-face {\n\t\t\tfont-family: \'Plaster\';\n\t\t\tsrc: url(\'path-to-plaster-font-file\');\n\t\t}\n\t\t@font-face {\n\t\t\tfont-family: \'Engagement\';\n\t\t\tsrc: url(\'path-to-engagement-font-file\');\n\t\t}\n'
+      );
+    });
```

Comment:
i would ask you if you could make up 2 additional font names. We have Plaster and Engagement, used in style and object props. If we had 4 names instead of 2, we could be sure that everyone is captured.